### ISSUE TITLE
docker ps -h: --no-trunc incorrectly documented

### ISSUE THREAD

**Issue by Annihilannic** [docker/docker#5545]
Surely it should read "--no-trunc=true: Don't truncate output" or "--no-trunc=false: Truncate output (default behaviour)".

Similar for "--quiet=false".  I presume it was intended to show the alternatives (either "-q" or "--quiet=false"), however the textual description is misleading.

I think a better format would be  "-q, --quiet=\<true|false\>: whether to only display numeric IDs", or something along those lines.

```
anni@helium:~/src/Redis$ docker ps -h

Usage: docker ps [OPTIONS]

List containers

  -a, --all=false: Show all containers. Only running containers are shown by default.
  --before="": Show only container created before Id or Name, include non-running ones.
  -l, --latest=false: Show only the latest created container, include non-running ones.
  -n=-1: Show n last created containers, include non-running ones.
  --no-trunc=false: Don't truncate output
  -q, --quiet=false: Only display numeric IDs
  -s, --size=false: Display sizes
  --since="": Show only containers created since Id or Name, include non-running ones.
anni@helium:~/src/Redis$
anni@helium:~/src/Redis$ docker version
Client version: 0.10.0
Client API version: 1.10
Go version (client): go1.2.1
Git commit (client): dc9c28f
Server version: 0.10.0
Server API version: 1.10
Git commit (server): dc9c28f
Go version (server): go1.2.1
Last stable version: 0.10.0
anni@helium:~/src/Redis$
```

**Comment by amouat** [comment]
I don't think it's correct to call `--no-trunc` a double negative. It only appears this way due to the documentation; it would not be problematic if #9406 was implemented.

I would avoid default true flags at all costs. In my opinion they are *extremely* confusing, see #10517.

**Comment by MHBauer** [comment]
With #18082 merged, is this still relevant? #18797 seems to be a separate and different issue. #5764 seems to pretty definitively resolve this one in favor of not renaming or providing alternative flags.

**Comment by thaJeztah** [comment]
Yes, I think this can be closed; the output has changed and is a lot less confusing;

```
Usage:	docker ps [OPTIONS]

List containers

  -a, --all          Show all containers (default shows just running)
  -f, --filter=[]    Filter output based on conditions provided
  --format           Pretty-print containers using a Go template
  --help             Print usage
  -l, --latest       Show the latest created container (includes all states)
  -n=-1              Show n last created containers (includes all states)
  --no-trunc         Don't truncate output
  -q, --quiet        Only display numeric IDs
  -s, --size         Display total file sizes
```

I'll close this issue
